core/scripts/cli/sponsor/listMarkets.js

```diff
@@ -1,15 +1,17 @@
 const inquirer = require("inquirer");
 const style = require("../textStyle");
+const showMarketDetails = require("./showMarketDetails");
 
 const listMarkets = async (web3, artifacts) => {
   style.spinnerReadingContracts.start();
   const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
-  const Governor = artifacts.require("Governor");
   const Registry = artifacts.require("Registry");
 
-  const governorAddress = (await Governor.deployed()).address;
+  const SyntheticToken = artifacts.require("SyntheticToken");
+  const Governor = artifacts.require("Governor");
```

Comment:
Tiny nit: Because you only use the address you strictly dont need to create an instance of it and call the async function below. You can just go `Govenor.address`

Reply:
Nice, thanks!